bluemira/codes/_freecadapi.py

```diff
@@ -1084,8 +1269,17 @@ def save_as_STP(shapes, filename="test", scale=1):
         Full path filename of the STP assembly
     scale: float (default 1)
         The scale in which to save the Shape objects
+
+    Notes
+    -----
+    This uses the legacy method to save to STP files.
+    It doesnt require freecad documents but also doesnt allow much customisation
+    The scale of Part is in mm by default therefore we scale by 1000 to convert
+    to metres.
```

Comment:
What are the pros and cons here? I vaguely remember us wanting to avoid FreeCAD documents, for some reason, but perhaps that was mostly for viewing CAD, not sure.

Reply:
We already use FreeCAD documents in the CAD viewer. The document is initialised in the viewer we just dont use them when creating our CAD. The FreeCAD document interface isnt very pythonic, its a straight conversion from C++ and also is another layer of abstraction, its I think why we use `Part` directly.

Possibly there could be some issues when running on something without a display but I don't think so as we explicitly initialise without a GUI, and there are probably other errors we havent dealt with in that area. 

In any event the tests pass here when there definitely isnt a display.